fab/fabfile.py

```diff
@@ -948,6 +950,19 @@ def _migrate():
 
 
 @roles(ROLES_DB_ONLY)
+def _migrations_exist():
+    """
+    Check if there exists database migrations to run
+    """
+    _require_target()
+    with cd(env.code_root):
+        n_migrations = int(sudo(
+            '%(virtualenv_root)s/bin/python manage.py migrate --list | grep "\[ ]" | wc -l' % env)
+        )
+        return n_migrations > 0
```

Comment:
Not quite sure how to implement in this context, but FYI, here's how django checks for un-run migrations:
https://github.com/django/django/blob/master/django/core/management/commands/runserver.py#L157-L178
```python
def _migrations_exist():
    executor = MigrationExecutor(connections[DEFAULT_DB_ALIAS])
    return executor.migration_plan(executor.loader.graph.leaf_nodes())
```